### PR TITLE
Feature/wall auto clip

### PR DESCRIPTION
**Pull request by Frocoa**
Automatizes the wall clipping when the view is rotated.

### CODE REVIEW

**Comment by Mortup** [comment]
Works in north and south orientations only :(

File: Assets/CoreEngine/Controller/IsometricTransform/WallTransformer.cs [hunk 1]
```diff
@@ -142,5 +142,41 @@ public static class WallTransformer {
...
+                        return new Vector2Int(wallCoords.x, wallCoords.y - 1);
+                    }
+                    else {
+                        return new Vector2Int(wallCoords.x - 1, wallCoords.y);
+                    }
+    
+            }
+
+
+            Debug.Log(OrientationManager.currentOrientation);
```
Comment (Mortup) [inline Assets/CoreEngine/Controller/IsometricTransform/WallTransformer.cs]: This line should be deleted

File: Assets/CoreEngine/Controller/SpriteObservers/WallSpriteObserver.cs [hunk 2]
```diff
@@ -104,6 +105,7 @@ public class WallSpriteObserver : IWallObserver, IOrientationObserver {
 		}
 
 		public void BindLevel(Level level) {
+            this.level = level;
```
Comment (Mortup) [inline Assets/CoreEngine/Controller/SpriteObservers/WallSpriteObserver.cs]: Is the indentation wrong here or github is just showing it wrong?

**Review by Mortup — CHANGES_REQUESTED** [review]

File: Assets/CoreEngine/Controller/SpriteObservers/WallSpriteObserver.cs (hunk 2 above)
Comment (Frocoa) [reply Assets/CoreEngine/Controller/SpriteObservers/WallSpriteObserver.cs]: Github is just showing it wrong

File: Assets/CoreEngine/Controller/IsometricTransform/WallTransformer.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Frocoa) [reply Assets/CoreEngine/Controller/IsometricTransform/WallTransformer.cs]: Github is just showing it wrong

**Review by Mortup — APPROVED** [review]

**Pull request merged**